components/transaction/NewTransactionForm.module.css

```diff
@@ -0,0 +1,14 @@
+.usercard {
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+input.paymentCardAmountInput {
```

Comment:
Why the input selector here?

Reply:
picocss (the css thingy used here) has a margin on all inputs, I needed to make my selector more specific than theirs to make it override:
![image](https://user-images.githubusercontent.com/10669773/216818544-c530570b-badc-45d7-bfe4-059b767af3ba.png)
or that's at least how I understand it